skypy/halo/tests/test_mass.py

```diff
@@ -0,0 +1,25 @@
+# import numpy as np
+import scipy.stats
+# import pytest
+
+import skypy.halo.mass as mass
+
+# def test_press_schechter():
+
+# Test that sampling corresponds to sampling from the right pdf.
+# For this, we sample an array of luminosities for redshift z = 1.0 and we
+# compare it to the corresponding cdf.
+
+# sample = mass.press_schechter(1.0, 1e9, size=1000)
+# p_value = scipy.stats.kstest(sample, calc_cdf)[1]
+# assert p_value >= 0.01
```

Comment:
I suggest a test similar to the kstest in test_herbel_luminosities comparing the Press-Schechter samples against the analytic cdf